src/components/SideMenu/styled.elements.ts

```diff
@@ -131,7 +132,7 @@ export const MenuItemLabel = styled.label.attrs((props: MenuItemLabelType) => ({
   padding: 12.5px ${props => props.theme.spacing.unit * 4}px;
   ${({ depth, type, theme }) =>
     (type === 'section' && depth > 1 && 'padding-left: ' + theme.spacing.unit * 8 + 'px;') || ''}
-  display: flex;
+  display: ${props => props.cssDisplay ? props.cssDisplay : 'flex'};
```

Comment:
I'd match the coding style from the similar code above and below and use parenthesis:
```suggestion
  display: ${props => (props.cssDisplay ? props.cssDisplay : 'flex')};
```